apps/ledger-live-desktop/package.json

```diff
@@ -59,6 +59,7 @@
     "@ledgerhq/logs": "workspace:^",
     "@ledgerhq/react-ui": "workspace:^",
     "@ledgerhq/types-cryptoassets": "workspace:^",
+    "@ledgerhq/types-devices": "workspace:^6.22.1",
```

Comment:
`"@ledgerhq/types-devices"` has already been added, just below - and i think we should keep the `"workspace:^"` version ;)